lib/chef-acceptance/test_suite.rb

```diff
@@ -84,11 +109,8 @@ def generate
       File.write(File.join(acceptance_cookbook_dir, 'metadata.rb'),
```

Comment:
It would be cool if we can have some erb templates that we render here instead of storing the contents of the files as strings buried in the code. 

While doing that we should also think of adding some comments & a README.md to the files we generate to kick-start the user of this command.